app/Http/Controllers/Api/UsersController.php

```diff
@@ -53,10 +53,36 @@ public function __construct(Registrar $registrar, Manager $manager, UserReposito
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(UserRequest $request)
     {
-        // @TODO: temp for now, likely want to use repository and transform.
-        return User::all();
+        $user = User::find($request['id']);
```

Comment:
@weerd we can tackle that tomorrow when @ScottPaillant gets in but I think this unblocks you?